website/docs/r/lb_backend_address_pool.html.markdown

```diff
@@ -50,8 +50,8 @@ resource "azurerm_lb_backend_address_pool" "example" {
 The following arguments are supported:
 
 * `name` - (Required) Specifies the name of the Backend Address Pool.
-* `resource_group_name` - (Required) The name of the resource group in which to create the resource.
 * `loadbalancer_id` - (Required) The ID of the Load Balancer in which to create the Backend Address Pool.
+* `resource_group_name` - (Optional / **Deprecated**) The name of the resource group in which to create the resource. Changing this forces a new resource to be created.
```

Comment:
which means this is no longer forcenew

Reply:
This function "SchemaResourceGroupNameForceNewDeprecated" is only for lb_backend_address_pool_resource.go since [this resource](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/internal/services/network/lb_backend_address_pool_resource.go#L23) doesn't include "Update" in schema. So I have to keep forcenew here for this resource.